packages/server/.env.example

```diff
@@ -71,3 +71,8 @@ PORT=3000
 # GLOBAL_AGENT_HTTP_PROXY=CorporateHttpProxyUrl
 # GLOBAL_AGENT_HTTPS_PROXY=CorporateHttpsProxyUrl
 # GLOBAL_AGENT_NO_PROXY=ExceptionHostsToBypassProxyIfNeeded
+##AWS Secrets Manager
+# AWS_ACCESS_KEY_ID=<your-access-key>
+# AWS_SECRET_ACCESS_KEY=<your-secret-key>
+# USE_AWS_SECRETS_MANAGER=true
+# CACHE_CREDENTIALS=true
```

Comment:
All other 3rd party providers should provide caching: https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/data-key-caching.html#:~:text=In%20general%2C%20use%20data%20key,NET.

For security purpose, I prefer not to cache it in the application, but rather on your key vault side